mintapi/signIn.py

```diff
@@ -549,6 +570,10 @@ def set_mfa_method(driver, mfa_method):
     )
     mfa_result = list(mfa)[0]
     try:
+        mfa_token_select = driver.find_element_by_css_selector(
```

Comment:
I believe THIS is the correct change needed to do what you're expecting.

The key, though, is if you want to select "email" as the MFA method at the choose screen, you need to pass in `mfa-method=email` as an argument.

Reply:
Thank you for catching this.  I made the change.

Reply:
@tsdg112 you bet! I'll test this out again this afternoon and make sure we're good. Then I'll be good to go here!